src/components/Map.ts

```diff
@@ -114,6 +96,41 @@ export class Map extends Component<MapProps, MapState> {
         }
     }
 
+    private renderGoogleMap(): ReactElement<GoogleMapProps> | null {
+        try {
+            return createElement(GoogleMap,
+                {
+                    bootstrapURLKeys: { key: this.props.apiKey },
+                    center: this.state.center,
+                    defaultZoom: this.props.zoomLevel,
+                    onGoogleApiLoaded: this.handleOnGoogleApiLoaded,
+                    options: {
+                        draggable: this.props.optionDrag,
+                        fullscreenControl: false,
+                        mapTypeControl: this.props.optionMapControl,
+                        maxZoom: 20,
+                        minZoom: 2,
+                        minZoomOverride: true,
+                        resetBoundsOnResize: true,
+                        scrollwheel: this.props.optionScroll,
+                        streetViewControl: this.props.optionStreetView,
+                        zoomControl: this.props.optionZoomControl
+                    },
+                    resetBoundsOnResize: true,
+                    yesIWantToUseGoogleMapApiInternals: true
+                },
+                this.createMakers()
+            );
+
+        } catch (error) {
+            const alertMessage = `${ this.props.friendlyId } load failed, Check internet connection.${error.message}`;
+
```

Comment:
Remove blank line. Can actually inline alertMessage.

Reply:
I am not removing it because of separation of concerns. variable declarations from body of code